vendor/parser-extended/docstrings.ml

```diff
@@ -118,6 +118,11 @@ let add_docs_attrs docs attrs =
   in
   attrs
 
+(** {!add_docs_attrs} but operate on [ext_attrs]. *)
+let add_docs_attrs' docs attrs' =
+  let open Parsetree in
+  { attrs' with attrs_before = add_docs_attrs docs attrs'.attrs_before }
```

Comment:
`attrs_before` is for short-syntax attributes ? I'm not sure it makes sense to put docstrings there.

Reply:
It is extracted afterward anyways, it does not make a difference, except a moral one. I'm fine either way.

Reply:
I want this to change. The intent behind the code is always important and will be interpreted by future contributors.